### PR TITLE
- completed task 1

### PR DESCRIPTION
**Pull request by PashaShevchuk**
- created component Footer;
- created component UserCard.

### CODE REVIEW

File: src/components/user-card/UserCard.js [hunk 1]
```diff
@@ -15,3 +15,24 @@ import React from 'react';
...
 //  - блок h4 с классом card-title, он должен вывести имя и фамилию (first_name, last_name)
 //  - блок div c классом "card-text", содержащий 2 div блока для поля email и поля address
+
+export const UserCard = (props) => {
+    const {user} = props;
+    if (!user) return null;
+    const {first_name, last_name, email, address, _links} = user;
+    return (
+        <div className='may-user-card card'>
+            <img src = {_links.avatar.href} className='may-user-card-avatar rounded-circle' alt='user photo'/>
```
Comment (taras1313) [inline src/components/user-card/UserCard.js]: you can do deeper destructuring
    const {first_name, last_name, email, address, _links : {avatar: { href }}} = user;

and then write just like this
        ```    user photo ```

it is not a 'must have'
but again, that is just looking a bit better

File: src/components/footer/Footer.js [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
+//   второй будет иметь класс "may-footer-social-media-wrapper", в него будут вложены 3 блока img в соответствии с socialMediaIcons
+//   каждый img должен иметь класс "may-footer-social-media-icon" и в пропсу src должен быть записан src из соответствующего объекта в socialMediaIcons массиве
+//   и в пропсу alt должен быть записан alt из соответствующего объекта в socialMediaIcons массиве
 
+export const Footer = () => {
+    return (
+        <div className='may-footer'>
+            <div className='may-footer-content'>
+                <div>Some text about authors</div>
+                <div className='may-footer-social-media-wrapper'>
```
Comment (taras1313) [inline src/components/footer/Footer.js:21]: you haven't covered how to repeat components, that are similar (like your img below)
but if you want, you could read some info about: how to create lists in react using map function

**Comment by taras1313** [comment]
good job

File: src/components/user-card/UserCard.js (hunk 1 above)
Comment (PashaShevchuk) [reply src/components/user-card/UserCard.js]: I corrected it

File: src/components/footer/Footer.js (hunk 2 above)
Comment (PashaShevchuk) [reply src/components/footer/Footer.js:21]: Did this with map function

**Pull request closed without merging**